dask/bytes/tests/test_s3.py

```diff
@@ -193,32 +201,32 @@ def test_getsize(fmt):
 
 def test_modification_time_read_bytes():
     with s3_context('compress', files) as s3:
-        _, a = read_bytes('compress/test/accounts.*', s3=s3)
-        _, b = read_bytes('compress/test/accounts.*', s3=s3)
+        _, a = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
+        _, b = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
```

Comment:
Am I in conflict with myself?  Probably.  Nothing here has an obvious answer.  This is all complex and its our job to find a nice compromise that is minimally incorrect.

This function feels more internal to me.  Functions like `read_text` and `read_csv` include keywords both for formats and for locations.  This function is only about location, it feels wrong to have a `storage_options` keyword for it just as it would feel odd to have a `csv_options` keyword in `read_csv`.